python/sglang/test/runners.py

```diff
@@ -292,8 +308,8 @@ def __init__(
             tp_size=tp_size,
             dtype=get_dtype_str(torch_dtype),
             port=port,
-            mem_fraction_static=mem_fraction_static,
-            trust_remote_code=False,
+            mem_fraction_static=0.65,
```

Comment:
Why not using the input argument `mem_fraction_static`?

Reply:
Yes this should be fixed